app/managers.py

```diff
@@ -0,0 +1,36 @@
+import sqlite3
+
+from app.models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema")
+        self.table_name = "Actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} "
+            f"(first_name, last_name) VALUES (?, ?)", (first_name, last_name)
+        )
+        self._connection.commit()
+
+    def all(self) -> list[Actor]:
+        actor_cursor = self._connection.execute(
+            f"SELECT * FROM {self.table_name}"
+        )
+        self._connection.commit()
```

Comment:
The `commit` operation is unnecessary in the `all` method since this method only performs a read operation. Consider removing this line to avoid confusion and unnecessary database operations.